.dockerignore

```
@@ -0,0 +1,3 @@
+* 
+
+entrypoint.sh
```

Since it's an explicit whitelist, I think this should be ...

```suggestion
!entrypoint.sh
```